### PR TITLE
chore(dependabot): enable for pip

### PR DESCRIPTION
**Pull request by st3iny**

### CODE REVIEW

**Review by ChristophWurst — APPROVED** [review]

File: .github/dependabot.yml [hunk 1]
```diff
@@ -6,3 +6,9 @@ updates:
       interval: weekly
       time: "06:00"
     open-pull-requests-limit: 10
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      time: "06:00"
+    open-pull-requests-limit: 10
```
Comment (nickvergessen) [inline .github/dependabot.yml:14]:
```suggestion
  - package-ecosystem: "pip"
    directory: "/"
    schedule:
      interval: "weekly"
      time: "06:00"
    target-branch: master
    open-pull-requests-limit: 10
  - package-ecosystem: "pip"
    directory: "/"
    schedule:
      interval: "weekly"
      time: "06:00"
    target-branch: stable26
    open-pull-requests-limit: 10
  - package-ecosystem: "pip"
    directory: "/"
    schedule:
      interval: "weekly"
      time: "06:00"
    target-branch: stable25
    open-pull-requests-limit: 10
  - package-ecosystem: "pip"
    directory: "/"
    schedule:
      interval: "weekly"
      time: "06:00"
    target-branch: stable24
    open-pull-requests-limit: 10
  - package-ecosystem: "pip"
    directory: "/"
    schedule:
      interval: "weekly"
      time: "06:00"
    target-branch: stable23
    open-pull-requests-limit: 10
```

**Review by nickvergessen — CHANGES_REQUESTED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (nickvergessen) [reply .github/dependabot.yml:14]: Or we want to backport them manually or not update them anymore on older versions?

Comment (ChristophWurst) [reply .github/dependabot.yml]: I would not bother with an update unless something is broken or vulnerable

**Review by nickvergessen — APPROVED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (ChristophWurst) [reply .github/dependabot.yml]: Can we remove the bumps for stable branches? The number of open PRs is insane

**Pull request merged**